Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/v1/conversion_generated.go

```diff
@@ -2958,8 +2958,22 @@ func autoConvert_api_SecurityContextConstraints_To_v1_SecurityContextConstraints
 	} else {
 		out.AllowedCapabilities = nil
 	}
-	out.AllowHostDirVolumePlugin = in.AllowHostDirVolumePlugin
-	out.AllowEmptyDirVolumePlugin = &in.AllowEmptyDirVolumePlugin
+
+	if in.Volumes != nil {
+		out.Volumes = make([]FSType, len(in.Volumes))
+		for i := range in.Volumes {
+			out.Volumes[i] = FSType(in.Volumes[i])
+		}
+	} else {
+		out.Volumes = nil
+	}
+
+	// set the Allow* fields based on the existence in the volume slice
+	volumeStringSet := fsTypeToStringSet(out.Volumes)
```

Comment:
move these up into the loop through the volume types.. a switch inside the loop with cases for hostpath/emptydir saves you a stringset construction. @smarterclayton, can we drop `AllowEmptyDirVolumePlugin`?